schema/avh.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/ARM-software/avhclient/main/schema/avh.schema.json",
```

Comment:
In this case, should we use the schema version for `v0.1.3` instead from `main`?

Reply:
This cannot work because the v0.1.3 tag does not contain this schema.